feature-configs/cn-ran-overlays/README.md

```diff
@@ -0,0 +1,25 @@
+# cn-ran-overlays
+
+This folder contains an example configurations for the RAN sites.
+Current example is focused on DU. CU profile can be built by
+- Replacing PTP by NTP (Work in progress)
+- Using stock kernel instead of RT kernel
+- Skipping FEC (Work in progress)
+
+
+## Overview
+
+The [`all-in-one`](all-in-one) directory contains the Kustomize profile for deployment of DU integration features, namely:
+- SCTP MachineConfig patch
+- Performance addon operator and DU performance profile
+- PTP operator and slave profile
+- SR-IOV operator and associated profiles
+
+## Deployment
+
+1. Make sure your nodes are labeled as required and MCP is created. If your cluster is brand new, this can be done by cnf-features-deploy/hack/setup-test-cluster.sh
+2. Deploy using the usual project tools:
+
+  `FEATURES_ENVIRONMENT=cn-ran-overlays FEATURES=all-in-one make feature-deploy`
```

Comment:
It makes sense to disconnect from the toolery here. 
Given the current approach of going with a single, non feature centric, profile, I'd specify how to run kustomize to do the deployment, and then add a note saying it can be done also via the makefile rule.

Reply:
+1